.devcontainer/Dockerfile

```diff
@@ -149,6 +149,8 @@ ENV PATH=$PATH:$HOME/.meteor/
 # Copy source dir
 USER root
 
+RUN echo "export PATH=$PATH" > /etc/environment
```

Comment:
Why replace all of environment?

Reply:
you are right, I fixed that. Also the $HOME-var was invalid for root.